crates/ra_editor/src/extend_selection.rs

```diff
@@ -10,7 +10,7 @@ pub fn extend_selection(file: &File, range: TextRange) -> Option<TextRange> {
     extend(syntax.borrowed(), range)
 }
 
-pub(crate) fn extend(root: SyntaxNodeRef, range: TextRange) -> Option<TextRange> {
+pub fn extend(root: SyntaxNodeRef, range: TextRange) -> Option<TextRange> {
```

Comment:
why are we making this public?

Reply:
Oops that's left over from an earlier iteration.

Reply:
I'll fix it tomorrow.